pkgs/top-level/python-packages.nix

```
@@ -10324,6 +10326,8 @@ in {
 
   sphinx-navtree = callPackage ../development/python-modules/sphinx-navtree { };
 
+  sphinx_mdinclude = callPackage ../development/python-modules/sphinx_mdinclude { };
```

```suggestion
  sphinx-mdinclude = callPackage ../development/python-modules/sphinx-mdinclude { };
```
New package names get normalized according to the docs.